docs/node/run-your-node/keymanager-node.md

```diff
@@ -0,0 +1,149 @@
+# Key Manager Node
+
+:::info
+
+These instructions are for setting up a _key manager_ node. Key manager nodes run a special runtime that provides confidentiality to other ParaTimes. If you want to run a _validator_ node instead, see the [instructions for running a validator node](validator-node/README.md). Similarly, if you want to run a _ParaTime_ node instead, see the [instructions for running a ParaTime node](paratime-node.mdx).
+
+:::
+
+:::tip
+
+At time of writing the key manager ParaTime is deployed only on the Testnet to support Cipher and Sapphire ParaTimes, and limited to be run by trustworthy partners.
+
+:::
+
+This guide will cover setting up your key manager node for the Oasis Network. This guide assumes some basic knowledge on the use of command line tools.
+
+## Prerequisites
+
+Before following this guide, make sure you've followed the [Prerequisites](prerequisites) and [Run a Non-validator Node](non-validator-node.md) sections and have:
+
+* Oasis Node binary installed on your system and a dedicated non-root user that will run your Oasis Node.
+* The chosen top-level `/node/` working directory prepared (feel free to name it as you wish, e.g. `/srv/oasis/`) with:
+  * `etc`: This will store the node configuration and genesis file.
+  * `data`: This will store the data directory needed by Oasis Node, including your node identity and the blockchain state. The directory permissions should be `rwx------`.
+  * `bin`: This will store binaries needed by Oasis Node for running the ParaTimes.
+  * `runtimes`: This will store the ParaTime bundles.
+* Downloaded or compiled the correct versions of everything according to [Mainnet Network Parameters](../mainnet/README.md):
+  * The genesis file copied to `/node/etc/genesis.json`.
+  * The binaries needed by Oasis Node for running the ParaTimes copied to `/node/bin/`.
+  * The key manager ParaTime bundle (`.orc` extension) copied to `/node/runtimes/`.
+* Initialized a new node and updated your entity registration by following the [Register a New Entity or Update Your Entity Registration](paratime-node.mdx#register-a-new-entity-or-update-your-entity-registration) instructions.
+  * The entity descriptor file copied to `/node/etc/entity.json`.
+
+:::tip
+
+Reading the rest of the [validator node setup instructions](validator-node/README.md) and [ParaTime node setup instructions](validator-node/README.md) may also be useful.
```

Comment:
Both links point to validator node setup.

Reply:
Oops, good catch.